server/features/EmailInbox/EmailInbox.ts

```diff
@@ -38,7 +40,7 @@ export async function configureEmailInboxes(): Promise<void> {
 			tlsOptions: {
 				rejectUnauthorized: false,
 			},
-			// debug: (...args: any[]): void => console.log(...args),
+			// debug: (...args: any[]): void => logger.debug(args),
```

Comment:
```suggestion
```

Do we need this comment?

Reply:
I mean, it's useful for local development, but leave it there mainly to let others know it's possible haha